pulp_ansible/app/downloaders.py

```diff
@@ -128,6 +128,7 @@ async def _run_with_additional_headers(self, headers):
             DownloadResult: Contains information about the result. See the DownloadResult docs for
                  more information.
         """
+        await asyncio.sleep(0.2)
```

Comment:
@newswangerd do you think 0.2 is good?
ref: https://hackmd.io/@fao89/rate_limiting

Reply:
Sorry I'm late to the party. I would have suggested something more like 0.5. The rationale being that we know the limit is 15 requests per second and concurrent connections defaults to 10, so that would give us a theoretical upper bound of 20 requests per second, assuming all requests are instant (which they aren't). That should roughly guarantee that nobody runs into this issue.

0.2 is probably fine though.